Apps/WebClient/src/ClientApp/src/store/modules/user/user.ts

```diff
@@ -10,6 +11,7 @@ import { mutations } from "./mutations";
 export const state: UserState = {
     statusMessage: "",
     user: new User(),
+    patientData: new PatientData(),
     error: false,
     status: LoadStatus.LOADED,
```

Comment:
status None since is not initialized

Reply:
> Can we display the label/value in one line only? (not wrapped)

@tiagograf-AOT can you elaborate on this? Is this comment for a specific line of code?

Reply:
weird, how can our current code work even with this flag as it is always there? Do we use this status to check at all?

Reply:
fixed code, it seems to work well as well with NONE LoadStatus